webapp/src/modules/mapBuilder.js

```diff
@@ -261,7 +258,7 @@ export default class MapBuilder {
   }
 
   startRefreshTimer(interval, bounds, mcg) {
-    if (this._refreshId && this._ongoingTrip) {
+    if (this._refreshId) {
```

Comment:
@rgalanakis This variable was probably in the works and wasn't cleaned up when it was removed.